app/views/pages/home.html.erb

```diff
@@ -1,9 +1,19 @@
 <div class="container col-8">
   <div class="d-flex align-items-center justify-content-between">
-    <h1>Find a Venue</h1>
-    <%= link_to "Venues", venues_path, class: "btn btn-primary" %>
+    <h1>Welcome to Find Venue</h1>
+  </div>
+
+
+  <div class="container">
+    <p1>Website created by Ronan, Jan and Jaro that will allow you to rent a venue.</p1>
```

Comment:
p1 should be just p here - only noticed after pull - we could probably update this page to look similar to the cards on Index too